### PR TITLE
Task/COOKS-373: add backend routes for community characteristics pie charts

### PR DESCRIPTION
**Pull request by nathanfranklin**
## Overview: ##

Add backend routes for community characteristics pie charts

## Related Jira tickets: ##

* [COOKS-373](https://jira.tacc.utexas.edu/browse/COOKS-373)

## Testing Steps: ##
1. Uncomment out line `fig.write_html("figure.html")` at end of demographics.py
2. Exec into protx container and run:
from protx.utils import demographics
result = demographics.get_age_race_pie_charts(area="county", geoid="48257")
3. View figure.html. ( i added a screenshot here).

## UI Photos:
Screenshot 2022-12-14 at 5 12 02 PM

## Notes: ##
I added text to https://jira.tacc.utexas.edu/browse/COOKS-377 so we can organize this better in the modal.  Like getting the font size correct and determining placement of texts. For example, the title "Community Characteristics" could be in the modal as well as the area info (e.g. "Community Characteristics of Travis County")

TODO:

- [x] Fix colors or put in note for  https://jira.tacc.utexas.edu/browse/COOKS-377

### CODE REVIEW

**Review by ryamase — APPROVED** [review]
LGTM :+1: 
Minor observation. I noticed the comment and logger text in the route both mention race/**gender** -- should they say race/**age**?

File: protx/utils/demographics.py [hunk 1]
```diff
@@ -2,10 +2,15 @@
 import json
 import numpy as np
 import pandas as pd
+from plotly.subplots import make_subplots
+import plotly.graph_objects as go
 from protx.utils.plotly_figures import timeseries_lineplot
 
 db_name = '/protx-data/cooks.db'
 
+six_colors_light_green_to_blue = ['#eff5d6', '#c6e8b0', '#8fcca1', '#62ad9c', '#3c7d8a', '#26547a']
```
Comment (taoteg) [inline protx/utils/demographics.py:11]: Can we consolidate these magic string values into one of the config file somewhere? 
I get a bad code smell mixing the styles(hexstring) values directly into the python code.

File: protx/utils/maltreatment.py [hunk 2]
```diff
@@ -6,7 +6,6 @@
 
 db_name = '/protx-data/cooks.db'
 
-# TODO.  this should be a mapping that doesn't use display names https://jira.tacc.utexas.edu/browse/COOKS-336
 maltrt_palette = {
```
Comment (taoteg) [inline protx/utils/maltreatment.py:9]: Same issue as demographics.
Its a bad code smell to hardcode these styles into the python logic.
We should  create a `styles.conf.py` config file (or something like that) for all visual style and design info to be tracked for use consistently across all the plot components (and the map).
Especially if we collapse the "many views" into a singular view that handles all the use cases for map + plots.

File: protx/utils/demographics.py (hunk 1 above)
Comment (taoteg) [reply protx/utils/demographics.py:11]: We should create a styles.conf.py config file (or something like that) for all visual style and design info to be tracked for use consistently across all the plot components (and the map).
Especially if we collapse the "many views" into a singular view that handles all the use cases for map + plots.

**Review by taoteg — APPROVED** [review]
Logic and code all looks good. I made some recommendations regarding hard-coded magic string values in the python logic. They do not impact the functionality of the PR, I can approve anyway if that concern is mitigated elsewhere.

File: protx/utils/maltreatment.py (hunk 2 above)
Comment (nathanfranklin) [reply protx/utils/maltreatment.py:9]: good idea 💯   will move to a conf file as suggested.

File: protx/utils/demographics.py (hunk 1 above)
Comment (nathanfranklin) [reply protx/utils/demographics.py:11]: > We should create a styles.conf.py config file (or something like that)

good idea 💯

**Comment by nathanfranklin** [comment]
> LGTM 👍 Minor observation. I noticed the comment and logger text in the route both mention race/**gender** -- should they say race/**age**?

good catch. fixed!

**Pull request merged**